deploy to cf changelog (#21488)
cloudflare/cloudflare-docs: src/content/changelog/workers/2025-04-08-deploy-to-cloudflare-button.mdx

---

> title: Set up and deploy a Workers application in seconds with Deploy to Cloudflare buttons

Reviewer: ```suggestion
title: Deploy a Workers application in seconds with one-click
```

maybe a shorter title?

description: You can now add a Deploy to Cloudflare button to your repository's README to help other developers set up and deploy your project. 
products:
  - workers
date: 2025-04-08 00:00:00 UTC

> hidden: true

Reviewer 2: ```suggestion
```
